### PR TITLE
[ENGINEERS-980] - Improvements

### PR DESCRIPTION
**Pull request by shashidhar85**

### CODE REVIEW

File: cypress/integration/2.1-rest.apis.spec.js [hunk 1]
```diff
@@ -4,7 +4,7 @@ import {
   updateProductStatus,
 } from '../support/availability-notify.apis'
 import { testSetup } from '../support/common/support'
```
Comment (Syed-Vtex) [inline cypress/integration/2.1-rest.apis.spec.js:6]: Drop the usage of testSetup and use loginViaCookies() which is available in support.js

Comment (Syed-Vtex) [reply cypress/integration/2.1-rest.apis.spec.js:6]: Please do this everywhere

File: cypress/support/commands.js [hunk 2]
```diff
@@ -45,15 +45,15 @@ Cypress.Commands.add('openProduct', (product, detailPage = false) => {
   }
 })
 
-Cypress.Commands.add('setavailabilitySubscribeId', (availabilityValue) => {
+Cypress.Commands.add('setavailabilitySubscribeId', availabilityValue => {
   const data = availabilityValue.filter(
-    (a) => a.email === 'saravananvenkatesan@bitcot.com'
+    a => a.email === 'saravananvenkatesan@bitcot.com'
```
Comment (Syed-Vtex) [inline cypress/support/commands.js:50]: read the mail id from VTEX_QE.json

File: cypress/support/availability-notify.js [hunk 3]
```diff
@@ -5,7 +5,7 @@ import {
 } from './availability-notify.graphql'
 import { updateRetry } from './common/support'
 import { getEmailContent } from './extract'
-import availabilityNotifySelectors from './availability-notify.selectors'
+import availabilityNotifySelectors from './selectors'
 import { MESSAGES } from './utils'
```
Comment (Syed-Vtex) [inline cypress/support/availability-notify.js:10]: is verify email works fine after we dropped the usage of gmail from availability notify

File: cy-runner.yml [hunk 4]
```diff
@@ -8,12 +8,6 @@ base:
     id: 2001459804
     domain: myvtex.com
     vtexIdUrl: https://vtexid.vtex.com.br/api/vtexid/pub/authenticate/default
-    deployCli:
-      enabled: true
-      git: https://github.com/vtex/toolbelt.git
-      branch: qe/cypress
-  twilio:
-    enabled: true
```
Comment (Syed-Vtex) [inline cy-runner.yml:16]: update jira board

File: cypress/support/commands.js (hunk 2 above)
Comment (shashidhar85) [reply cypress/support/commands.js:50]: ok

File: cypress/support/availability-notify.js (hunk 3 above)
Comment (shashidhar85) [reply cypress/support/availability-notify.js:10]: yes

File: cypress/support/outputvalidation.js [hunk 5]
```diff
@@ -1,11 +1,11 @@
 export default {
   data1: {
     skuId: '880223',
-    name: 'saravanan',
-    email: 'saravananvenkatesan@bitcot.com',
+    name: 'shashi',
+    email: 'shashi@bitcot.com',
```
Comment (Syed-Vtex) [inline cypress/support/outputvalidation.js:5]: here also it is hardcoded can u pls make it dynamic

File: cy-runner.yml [hunk 6]
```diff
@@ -75,28 +70,38 @@ strategy:
     stopOnFail: false
     parallel: true
     specs:
-      - cypress/integration/2.1-*
-      - cypress/integration/2.2-*
-  ui_testcase:
+      - cypress/integration/2.1-rest.apis.spec.js
+      - cypress/integration/2.2-graphql.spec.js
```
Comment (Syed-Vtex) [inline cy-runner.yml:74]: Good improvement

**Review by Syed-Vtex — APPROVED** [review]
LGTM

**Pull request merged**